src/components/settings-menu/DevSection.js

```diff
@@ -126,6 +131,29 @@ const DevSection = () => {
 
       <UserDevSection scrollEnabled={false} />
 
+      <ListItem
+        label={`‍⏩ Sync codepush, current: ${codePushVersion}`}
+        onPress={async () => {
```

Comment:
Would be cool to extract it into the `useCallback` just to follow the same pattern for callbacks in this component